### DISCUSSION TITLE
Windows 10 Start button too small. (Windows 11 23H2, ExplorerPatcher V22621.3527.65.5)

### DISCUSSION THREAD

**Discussion by IamaPineapple24** [valinet/ExplorerPatcher#3531] [Q&A]
![image](https://github.com/user-attachments/assets/2970e674-e4c4-401a-9771-ea33c1fd6879)
I don't have anything else to add to this, it seems pretty self explanatory but if you have any questions I'll answer best I can.

Edit: I believe this is what it should look like:
![image](https://github.com/user-attachments/assets/df4ebdad-a9d4-4124-9036-94863a0abd84)

**Comment by Amrsatrio** [comment]
The Start button icon of the taskbar in Windows 10 is like in the first image, there is nothing wrong.

The sizing in the second image is only present in 8 and 8.1.

**Reply by IamaPineapple24** [reply]
that makes sense- i couldn't find a texture specifically from 10 so i downloaded one from 8.1... as a side note- i don't remember the start button always being grey when you hover it. is that just me misremembering or is that an actual glitch in explorerpatcher?

![image](https://github.com/user-attachments/assets/ddd99a06-1a96-40fd-9f61-063bd4cedadc)
(as you can see my accent colour is red, it's only set on the taskbar for demonstration purposes, but the button is always like that)

**Reply by pyrates999** [reply]
how did you change your start menu button to the custom one you downloaded?

**Reply by IamaPineapple24** [reply]
Start11. It's a paid app- not really worth it though... unfortunately I only found out about ExplorerPatcher AFTER I paid for it lmao

**Reply by pyrates999** [reply]
Try installing [open shell](https://github.com/Open-Shell/Open-Shell-Menu) to change the start menu button.

**Reply by IamaPineapple24** [reply]
I thought they stopped development of that in 2010

**Reply by pyrates999** [reply]
That was classic shell that had development stopped on it.  Open shell is the replacement.

**Comment by milestprower92** [comment]
> ![image](https://private-user-images.githubusercontent.com/103263186/354837871-2970e674-e4c4-401a-9771-ea33c1fd6879.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MjI3NDU2MjEsIm5iZiI6MTcyMjc0NTMyMSwicGF0aCI6Ii8xMDMyNjMxODYvMzU0ODM3ODcxLTI5NzBlNjc0LWU0YzQtNDAxYS05NzcxLWVhMzNjMWZkNjg3OS5wbmc_WC1BbXotQWxnb3JpdGhtPUFXUzQtSE1BQy1TSEEyNTYmWC1BbXotQ3JlZGVudGlhbD1BS0lBVkNPRFlMU0E1M1BRSzRaQSUyRjIwMjQwODA0JTJGdXMtZWFzdC0xJTJGczMlMkZhd3M0X3JlcXVlc3QmWC1BbXotRGF0ZT0yMDI0MDgwNFQwNDIyMDFaJlgtQW16LUV4cGlyZXM9MzAwJlgtQW16LVNpZ25hdHVyZT1mYTY1YjhkOWE2ZDFkZTNmNDM2NGRkMmM1MmI3ZjVkYWFmNzIxOWJkYWIzM2M2YTdiMWI0OTk4YjU3MDk3NWNhJlgtQW16LVNpZ25lZEhlYWRlcnM9aG9zdCZhY3Rvcl9pZD0wJmtleV9pZD0wJnJlcG9faWQ9MCJ9.b5G61zZm4u0aKLL7f1dAvPv5cLId3Na4jsLQhqGgxOY) I don't have anything else to add to this, it seems pretty self explanatory but if you have any questions I'll answer best I can.
> 
> Edit: I believe this is what it should look like: ![image](https://private-user-images.githubusercontent.com/103263186/354837955-df4ebdad-a9d4-4124-9036-94863a0abd84.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MjI3NDU2MjEsIm5iZiI6MTcyMjc0NTMyMSwicGF0aCI6Ii8xMDMyNjMxODYvMzU0ODM3OTU1LWRmNGViZGFkLWE5ZDQtNDEyNC05MDM2LTk0ODYzYTBhYmQ4NC5wbmc_WC1BbXotQWxnb3JpdGhtPUFXUzQtSE1BQy1TSEEyNTYmWC1BbXotQ3JlZGVudGlhbD1BS0lBVkNPRFlMU0E1M1BRSzRaQSUyRjIwMjQwODA0JTJGdXMtZWFzdC0xJTJGczMlMkZhd3M0X3JlcXVlc3QmWC1BbXotRGF0ZT0yMDI0MDgwNFQwNDIyMDFaJlgtQW16LUV4cGlyZXM9MzAwJlgtQW16LVNpZ25hdHVyZT0wNTU1ZWY5ODU1NWIyMDliMmNmZTY2ZDRmNzE4ZmY4YzkzNWU2ODYzM2EzMjgxNmY4ODIzMmZkNDFkY2MwMzc3JlgtQW16LVNpZ25lZEhlYWRlcnM9aG9zdCZhY3Rvcl9pZD0wJmtleV9pZD0wJnJlcG9faWQ9MCJ9.Id9DXmZjBXnGs7KJEv2PwFiQqZMB5PkJkN4jVlD1SEI)

ExplorerPatcher does not modify the start menu icon/size like StartIsBack that's why it looks small, since that is how microsoft made it for the windows 10 taskbar back in 2015

**Comment by pyrates999** [comment]
also see here: https://github.com/valinet/ExplorerPatcher/wiki/All-features#start-button-style

and see this comment for how you can change the start menu button if you wish to: https://github.com/valinet/ExplorerPatcher/discussions/436#discussioncomment-6448247

**Reply by milestprower92** [reply]
The new reimplemented taskbar now changes the start, search and taskview buttons to match with the windows 11 taskbar
